src/App.vue

```diff
@@ -1,10 +1,17 @@
 <template>
   <div id="app">
     <h1>BridgeIT</h1>
+    <PageHeader />
   </div>
 </template>
 <script>
-export default {};
+import PageHeader from '@/components/PageHeader';
+
```

Comment:
Każdy komponent będzie miał swój prywatny folder dedykowany danemu komponentowi. Nie jest to błąd a informacja na przyszłość